common/src/main/java/com/revenuecat/purchases/common/purchaseExtensions.kt

```diff
@@ -5,21 +5,19 @@ import com.revenuecat.purchases.strings.BillingStrings
 import java.util.ArrayList
 
 fun Purchase.toHumanReadableDescription() =
-    "skus: ${
-        this.skus.joinToString(prefix = "[", postfix = "]")
+    "productIds: ${
+        this.products.joinToString(prefix = "[", postfix = "]")
     }, orderId: ${this.orderId}, purchaseToken: ${this.purchaseToken}"
 
 /**
  * **Important**: until multi-line subscriptions are released,
  * we assume that there will be only a single sku in the purchase.
```

Comment:
Hmmm we should probably look into this to see if it's released? 🤔 I think last time we checked it wasn't live right?

Reply:
yeahh i'll make a ticket to remember this